### DISCUSSION TITLE
Reports

### DISCUSSION THREAD

**Discussion by Stefandro90** [Leantime/leantime#525] [Q&A]
Excuse me, does anyone understand the graphics under the heading reports?

Thank you :)

**Comment by marcelfolaron** [comment]
Hey, 

We need to add an article on those reports in our documentation. There are multiple graphs:

1. The top 4 blue squares are primarily for hourly tracking. If you set a budget for hours and have team members add their hours, you will see updated metrics in there
2. If you run sprints (scrum) your will see a sprint burn down. How many tasks have been completed and what the ideal distribution would be. 
3. The Cumulative Flow graph is a stacked chart of your done,in progress and open todos. In a perfect world everything would be red at the beginning of a project, then green, yellow, red for the duration and at the end everything should be green. 

Let me know if that helps.
